config/additional-supervisord.conf

```diff
@@ -19,3 +19,14 @@ stderr_logfile = /var/log/clamd.log
 stderr_logfile_maxbytes = 5000000
 stderr_logfile_backups = 3
 stopsignal = INT
+
+[program:cron]
```

Comment:
Isn't there an option to `supervisord` that allows it to do e.g. hourly actions without cron? Events or something?